tests/unit/types/arrays/mixins/test_eval_class.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pytest
+
+from jina import DocumentArray, Document
+
+
+@pytest.mark.parametrize(
+    'metric_fn, kwargs',
+    [
+        ('r_precision', {}),
+        ('precision_at_k', {}),
+        ('hit_at_k', {}),
+        ('average_precision', {}),
+        ('reciprocal_rank', {}),
+        ('recall_at_k', {'max_rel': 9}),
+        ('f1_score_at_k', {'max_rel': 9}),
+        ('ndcg_at_k', {}),
+    ],
+)
+def test_eval_mixin_perfect_match(metric_fn, kwargs):
```

Comment:
@hanxiao I do not understand how is here NDCG evaluated? How can I pass relevance information of the different groundtruth elements?